spec/parse_date_spec.rb

```diff
@@ -5,6 +5,79 @@
     expect(ParseDate::VERSION).not_to be nil
   end
 
+  describe '.parse_range' do
+    context 'when input is parseable' do
+      # single value
+      { # string to parse as key, expected result as value
+        '12/25/00' => [2000],
+        '5-1-25' => [1925],
+        '1666 B.C.' => [-1666],
+        '-914' => [-914],
+        '[c1926]' => [1926],
+        'ca. 1558' => [1558],
+      }.each do |example, expected|
+        it "array of single value #{expected} for '#{example}'" do
+          expect(ParseDate.parse_range(example)).to eq expected
+        end
+      end
+
+      # multiple values
+      { # string to parse as key, expected result as value
+        '195-' => (1950..1959).to_a,
+        '199u' => (1990..1999).to_a,
+        '197?' => (1970..1979).to_a,
+        '196x' => (1960..1969).to_a,
+        '18th century CE' => (1700..1799).to_a,
+        '17uu' => (1700..1799).to_a,
+        'between 1694 and 1799' => (1694..1799).to_a,
+        'between 1 and 5' => (1..5).to_a,
+        'between 300 and 150 B.C.' => (-300..-150).to_a,
+        '-5 - 3' => (-5..3).to_a,
+        '1496-1499' => (1496..1499).to_a,
+        '1750?-1867' => (1750..1867).to_a,
+        '17--?-18--?' => (1700..1899).to_a,
+        '1835 or 1836' => [1835, 1836].to_a,
```

Comment:
`.to_a` seems superfluous here, but def wouldn't hold up merge over that

Reply:
good 👀 - thanks.